src/api/public/apidocs-new/paths/search_request_id.yaml

```diff
@@ -0,0 +1,85 @@
+get:
+  summary: List request numbers that match a XPath condition.
+  description: |
+    Return a collection of request numbers that match a XPath condition.
+
+    This operation is the same as the one defined with [GET /search/request](#/Search/get_search_request) with the exception of the results returned.
+    While the former operation returns a list of request objects, this one return a list of request numbers. See example values of a succeeded request below.
+  security:
+    - basic_authentication: []
+  parameters:
+    - in: query
+      name: match
+      schema:
+        type: string
+      description: |
+        Expression based in XPath.
+
+        Not providing a value or providing a value of `*` will return all requests.
+
+        Available predicates are:
+          - Request fields: `@id` and `@creator`.
+          - State fields: `state/@name`, `state/@who`, and `state/@when`.
+          - Target fields: `target/@project` and `target/@package`.
+          - Source fields: `source/@project` and `source/@package`.
+          - Actions fields: `action/@type`, `action/grouped/@id`, `action/target/@project`, `action/target/@package`, `action/source/@project`, and `action/source/@package`.
+          - Reviews fields:
+            - General: `review/@when`, `review/@state`.
+            - Review by: `review/@by_user`, `review/@by_group`, `review/@by_project`, and `review/@by_package`.
+            - History: `review/history/@when`.
+          - History fields: `history/@when` and `history/@who`.
+      examples:
+        all:
+          summary: All
+          value: '*'
+        id:
+          summary: Request number
+          value: '@id=123456'
+        function_contains:
+          summary: Function contains
+          value: contains(@creator,'Igg')
+        combination:
+          summary: State name and date
+          value: /search/request?match=state/@name='review' and starts_with(state/@when,'2022-08-22')
+    - $ref: '../components/parameters/search_limit.yaml'
+    - $ref: '../components/parameters/search_offset.yaml'
```

Comment:
We should also have the `withhistory` and `withfullhistory` here.

Reply:
Right. Added.